### DISCUSSION TITLE
Retire the mailing list in favor of GitHub Discussions

### DISCUSSION THREAD

**Discussion by stasm** [unicode-org/message-format-wg#135] [Ideas]
GitHub Discussions are [now available as a public beta](https://github.blog/changelog/2020-12-08-github-discussions-now-available-as-a-public-beta/). Should we give them a try and if we're happy with them, consider switching away from the mailing list?

**Comment by romulocintra** [comment]
I just converted this into a Discussion to test behaviour 👍🏿

**Reply by stasm** [reply]
Good idea, thanks!
